src/routes/auth/index.js

```diff
@@ -11,5 +10,10 @@ router.get('/verify_email/:token', authentication.emailVerification);
 router.post('/forgotPassword', authentication.forgotPassword);
 router.put('/resetPassword/:token', validate.resetPassValidator, authentication.resetPassword);
 router.post('/otp', authentication.checkotp);
-router.patch("/updatePassword",verifyToken, authentication.updatePassword)
+router.patch('/updatePassword', verifyToken, authentication.updatePassword);
+router.get('/check', validate.authorize(['admin', 'vendor', 'normal']), (req, res) => {
+  res.status(200).json({
+    message: 'You are authorized',
+  });
+});
```

Comment:
if validate.authorize works good it's fine but if we have a lot of condition to check from here the routes directory will become more complex as project keeps growing better to define all these conditions from middle directory and use it here as a middleware function. that's my suggestion.